hw/drivers/flash/spiflash/src/spiflash.c

```diff
@@ -876,6 +876,8 @@ spiflash_wait_ready(struct spiflash_dev *dev, uint32_t timeout_ms)
     spiflash_lock(dev);
 
     while (!spiflash_device_ready(dev)) {
+        /* If not ready let's give it 10ms */
+        os_time_delay(os_time_ms_to_ticks32(10));
         if (os_time_get() > exp_time) {
```

Comment:
I have a question about this 10msec delay here. The spiflash_wait_ready() function is called by called that does not do erases. Waiting 10msecs seems like it could seriously impact flash read/write times, especially if you are doing small reads and writes. Was this change added to the code? I think I would only do something like this when erasing as opposed to reading/writing flash. Looks like there is a change below to that effect...